src/Features/Core/Portable/CodeFixes/Service/CodeFixService.FixAllDiagnosticProvider.cs

```diff
@@ -68,7 +68,7 @@ public override async Task<IEnumerable<Diagnostic>> GetAllDiagnosticsAsync(Proje
             // Get all diagnostics for the entire project, including document diagnostics.
             var service = project.Solution.Services.GetRequiredService<IDiagnosticAnalyzerService>();
             var diagnostics = Filter(await service.GetDiagnosticsForIdsAsync(
-                project, documentId: null, _diagnosticIds, shouldIncludeAnalyzer: null, includeLocalDocumentDiagnostics: true, includeNonLocalDocumentDiagnostics: false, cancellationToken).ConfigureAwait(false));
+                project, documentId: null, _diagnosticIds, shouldIncludeAnalyzer: null, includeLocalDocumentDiagnostics: true, cancellationToken).ConfigureAwait(false));
```

Comment:
these changed.  but not in a way that should matter.

1. we're doing fix-all.  So we *are* running the analyzer fully to compute results.
2. this will now catch the case that the analyzer reports diagnostics for different files, having the fixer attempt to fix those as well.